### PR TITLE
Add document about connecting to localhost

### PR DESCRIPTION
**Pull request by fhiyo**
##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```
ansible 2.1.0.0
  config file =
  configured module search path = Default w/o overrides
```

##### SUMMARY

In "Your first commands" section of "Getting Started", the example instructs users to prepare some remote servers. But, as the first example to try ansible, it might be a good idea to add document how to ping to localhost.

If you write `localhost` or `127.0.0.1` in `/etc/ansible/hosts`, below error occurs.

```
localhost | UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh.",
    "unreachable": true
}
```

Indeed, you cannot confirm communication to localhost if you do not specify localhost with this way of writing in `/etc/ansible/hosts`:

```
[Local]
localhost ansible_connection=local
```

I think it is good to add description of the way of specify localhost, and ping.

### CODE REVIEW

**Comment by sivel** [comment]
I think the preferred way is actually to not supply localhost in your inventory at all, which gives you access to an implicitly added localhost, which also specifies the correct `ansible_python_interpreter` as well as use of the local connection.

**Comment by bcoca** [comment]
I would prefer note about implicit localhost vs this.

**Comment by fhiyo** [comment]
I added a description that localhost does not need to be written inventory, and what to do if you explicitly write to inventory.

File: docs/docsite/rst/intro_getting_started.rst [hunk 1]
```diff
@@ -107,6 +108,21 @@ explore what you can do with different modules, and to learn about the Ansible
...
+
+Example:
+
+.. code-block:: bash
+
+    $ ansible localhost -m ping
+
+Also, you can specify localhost explicitly by writing like this on your inventory file::
+
+    localhost ansible_connection=local
```
Comment (bcoca) [inline docs/docsite/rst/intro_getting_started.rst]: add ansible_python_interpreter

File: docs/docsite/rst/intro_getting_started.rst [hunk 2]
```diff
@@ -107,6 +108,21 @@ explore what you can do with different modules, and to learn about the Ansible
 also has powerful configuration management and deployment features.  There's more to
 explore, but you already have a fully working infrastructure!
 
+Tips
+
+Only the local server, you do not need to write on inventory file to specify the local server.
```
Comment (bcoca) [inline docs/docsite/rst/intro_getting_started.rst]: To reference the machine on which you are running Ansible (commonly known as the 'controller'), you don't need to specify it in the inventory as Ansible provides an 'implicit localhost'.

File: docs/docsite/rst/intro_getting_started.rst [hunk 3]
```diff
@@ -107,6 +108,21 @@ explore what you can do with different modules, and to learn about the Ansible
...
+All you have to do is typing "localhost" or "127.0.0.1" on where specifies a server name.
```
Comment (bcoca) [inline docs/docsite/rst/intro_getting_started.rst]: s/typing/type/ 

s/on where specifies/anywhere you need to specify/

**Review by dharmabumstead — APPROVED** [review]
Approved with edits.

**Comment by fhiyo** [comment]
Added ansible_python_interpreter. Is it okay?

**Review by dharmabumstead — APPROVED** [review]

**Comment by dharmabumstead** [comment]
Thanks @fhiyo!

**Pull request merged**